modules/gamescope.yml

```diff
@@ -0,0 +1,106 @@
+name: gamescope
+buildsystem: meson
+config-opts:
+  - -Dpipewire=enabled
+  - -Dforce_fallback_for=stb
+sources:
+  - type: git
+    url: https://github.com/ValveSoftware/gamescope.git
+    commit: 7aef264dfcf81c5f303422d820b7e1ecf81568f3
+    tag: 3.11.51
+    x-checker-data:
+      type: git
+      tag-pattern: ^([\d.]+)$
+      is-main-source: true
+  - type: shell
+    commands:
+      - ln -s ${FLATPAK_DEST}/include/stb subprojects/stb
+      - install -Dm644 subprojects/packagefiles/stb/meson.build -t subprojects/stb/
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - /lib/*.a
+modules:
+  - name: stb
+    buildsystem: simple
+    build-commands:
+      - |
+        set -e
+        for f in *.h; do
+          install -Dm644 $f -t ${FLATPAK_DEST}/include/stb/
+        done
+    sources:
+      - type: archive
+        url: https://github.com/nothings/stb/archive/5736b15f7ea0ffb08dd38af21067c314d6a3aae9/stb-5736b15f7ea0ffb08dd38af21067c314d6a3aae9.tar.gz
+        sha256: d00921d49b06af62aa6bfb97c1b136bec661dd11dd4eecbcb0da1f6da7cedb4c
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/nothings/stb/commits?sha=master&per_page=1
+          version-query: .[].sha
+          timestamp-query: .[].commit.committer.date
+          url-query: '"https://github.com/nothings/stb/archive/" + $version + "/stb-"
+            + $version + ".tar.gz"'
+    cleanup:
+      - '*'
+      
+  - name: glm
+    buildsystem: cmake-ninja
+    no-make-install: true
+    post-install:
+      - install -d ${FLATPAK_DEST}/include
+      - cp -R glm ${FLATPAK_DEST}/include
+      - cp -R cmake/glm ${FLATPAK_DEST}/lib/cmake
+      - cp glm.pc.in ${FLATPAK_DEST}/lib/pkgconfig/glm.pc
```

Comment:
this pkgconfig will show `@VERSION@` when querying for `--modversion`

Reply:
Doesn't really matter, it's not something anyone would rely on. At least, I think. I'm not sure how to change the version anyways.

Reply:
It seems upstream, in their packaging hostility, refuses to include the necessary bits for this too:

https://github.com/g-truc/glm/pull/1039

So either I hardcode it, or leave it as-is. Neither solution is good.